package.json

```diff
@@ -13,7 +13,13 @@
     "dist-mb": "sass scss/bootstrap-mb.scss | tee | postcss --use autoprefixer --no-map > dist/bootstrap-mb.css",
     "dist-generic": "sass scss/bootstrap-generic.scss | tee | postcss --use autoprefixer --no-map > dist/bootstrap-generic.css",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook -o docs-build"
+    "build-storybook": "build-storybook -o docs-build",
+    "lint": "eslint --ext .ts,.tsx,.js,.jsx"
```

Comment:
I think this command might be missing a file path at the end.

With the hint of the workflow action that hangs forever, I tried running this command .
It finished finished in a second with no output which I found suspicious.
Running the command `yarn run lint .` (which translates to `eslint --ext .ts,.tsx,.js,.jsx .`) with the dot at the end hangs forever (that's similar to the command being run by the action).

Running `yarn run lint ./components/` produces the eslint output I expect with warnings and a recap:
<img width="895" alt="image" src="https://user-images.githubusercontent.com/6179856/170542783-949eb0a7-56a7-45b3-92a7-8463c61739cf.png">

Are there other directories we want to run eslint on? `stories` maybe?
In which case `eslint --ext .ts,.tsx,.js,.jsx  ./components ./stories` works nicely.